tests/data/rutos-rutx_rut360.json

```diff
@@ -392,6 +413,258 @@
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "2G_3G",
+                    "state_draw_graph": 1,
+                    "state_value": 28,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "GSM_WCDMA",
+                    "state_draw_graph": 1,
+                    "state_value": 29,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "2G_4G",
+                    "state_draw_graph": 1,
+                    "state_value": 30,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "GSM_LTE",
+                    "state_draw_graph": 1,
+                    "state_value": 31,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "3G_4G",
+                    "state_draw_graph": 1,
+                    "state_value": 32,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "WCDMA_LTE",
+                    "state_draw_graph": 1,
+                    "state_value": 33,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "3G_5G",
+                    "state_draw_graph": 1,
+                    "state_value": 34,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "WCDMA_NR5G",
+                    "state_draw_graph": 1,
+                    "state_value": 35,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "4G_5G",
+                    "state_draw_graph": 1,
+                    "state_value": 36,
+                    "state_generic_value": 0
+                },
+                {
+                    "state_name": "TELTONIKA-RUTX-MIB::ConnectionType",
+                    "state_descr": "LTE_NR5G",
+                    "state_draw_graph": 1,
+                    "state_value": 37,
+                    "state_generic_value": 0
+                },
+                {
```

Comment:
Seems that you hand-crafted the file, and made too many copy/paste below. Please use the script to regenerate all test-data for the OS. That's the best way to avoid mistakes.